src/array.js

```diff
@@ -22,14 +26,15 @@ if (!Array.from) {
         , C = typeof this == 'function' ? this : Array
         , aLen = arguments.length
         , mapfn = aLen > 1 ? arguments[1] : undefined
+        , thisArg = aLen > 2 ? arguments[2] : undefined
```

Comment:
I know `mapfn` assignment is written in the same way, but `arguments[2]` will be `undefined` anyway if `aLen <= 2`.